app/views/publish/courses/_course_length_field.html.erb

```diff
@@ -4,22 +4,17 @@
     id: form_object.errors.key?(:course_length) ? "course_length-error" : "course-length",
   }) do %>
     <%= f.govuk_radio_button(:course_length, "OneYear",
+      checked: @course.enrichments.last[:course_length] == "OneYear",
```

Comment:
@course.course_length ?

Reply:
@course.course_length works here, but not in courses/course_information/edit.html.erb

Reply:
Actually it has started working, I hate these tests...